packages/react-components/stories/tree-button-group.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { TreeButtonGroup } from '../lib';
+
+export default {
+  title: 'Tree Button Group',
+  component: TreeButtonGroup,
+} as Meta;
+
+export const SimpleTreeButtonGroup = (): React.ReactElement => {
```

Comment:
Make use of storybook's control addon to provide a UI to adjust the props. See https://storybook.js.org/docs/react/essentials/controls.